samples/custom_metrics.js

```diff
@@ -26,16 +26,24 @@ export default function () {
     let res = http.get("http://httpbin.org/");
     let passed = check(res, { "status is 200": (r) => r.status === 200 });
 
+
     // Add one for number of requests
     myCounter.add(1);
 
-    // Set max response time seen
+	console.log(myCounter.getName(), " is config ready")
```

Comment:
The indentation looks wrong here, uses a tab instead of the spaces used in the rest of the file. Besides, we probably don't need to demonstrate that every custom metric has this new method, a single example will probably be enough.